### ISSUE TITLE
Queryables: Use of $ref

### ISSUE THREAD

**Issue by m-mohr** [stac-api-extensions/filter#12]
In implementation I'm facing issues with the queryables and I'm wondering what the best practice should be: Exposing $ref's or derefernecing them?

The example in the README clearly uses $ref's, but in implementations (especially browser based) this leads to issues as a lot of requests may be invoked by this. So I'm wondering whether implementations should dereference (and cache) this. (I'd be in favor of this.)

Here's a related issue in MS PC: https://github.com/microsoft/planetary-computer-apis/issues/153

**Comment by philvarner** [comment]
Agree, in practice, the $refs should be dereferenced and a complete, standalone json schema should be returned.
